index.bs

```diff
@@ -8646,30 +8696,33 @@ a reference to the same object that the IDL value represents.
 </div>
 
 <div algorithm>
-    The <dfn export for="BufferSource">byte length</dfn> of a {{BufferSource}} |bufferSource| is
-    the value returned by the following steps:
+    The <dfn id=buffersource-byte-length export for="buffer source type">byte length</dfn> of a
+    [=buffer source type=] |bufferSource| is the value returned by the following steps:
```

Comment:
So for example, I think using the typedef here is more correct, because if |bufferSource| were a "buffer source type", then it would either be the type "DataView" or the type "ArrayBuffer" or the type .... It would not be any specific IDL object.